memo.rb

```diff
@@ -28,15 +39,12 @@ def save_memos(memos)
 end
 
 post '/memos' do
-  memos = load_memos
-  new_memo = { SecureRandom.uuid => { 'title' => params[:title], 'content' => params[:content] } }
-  memos.merge!(new_memo)
-  save_memos(memos)
+  make_memo(params[:title], params[:content])
```

Comment:
```suggestion
  create_memo(params[:title], params[:content])
```

makeよりもcreateの方が業界的にはよく使われますね